detect/synthetic-monitoring/playwright-checks/custom-dependencies.mdx

```diff
@@ -30,30 +32,45 @@ Checkly detects your package manager from your lock file:
 | `pnpm-lock.json`    | pnpm            |
 | `yarn.lock`         | yarn            |
 
-<Warning>
-**For pnpm users:** If you define the same dependency in both `dependencies` and `devDependencies`, use a custom `installCommand` in your `checkly.config.ts` to install all dependencies: `pnpm install`
-</Warning>
+To override the account-wide setting for a specific check suite, use the `installCommand` property in your `checkly.config.ts`:
```

Comment:
This might be a lack of context on my end, but what are we referring to here with "the account-wide setting"? 

Trying to understand it in contrast to the previous sentence a bit further up "You can change this account-wide setting to install only devDependencies in your [account settings](https://app.checklyhq.com/accounts/settings/account/playwright?_gl=1*48h0k3*_ga*MjExNjQ5OTUxNS4xNzU5MTUzMzIx*_ga_SYSCT5RC16*czE3NjYxNTY4NTkkbzM3JGcxJHQxNzY2MTU2ODc0JGo0NSRsMCRoMA..)."